### ISSUE TITLE
custom-style element appears to have a size limit

### ISSUE THREAD

**Issue by kirilledelman** [Polymer/polymer#3741]
This was a huge pain to narrow down.

This only seems to happen on Chrome. I have a relatively large `<style is="custom-style">` element that styles all the paper components via CSS mix-ins and a lot of custom stuff in my application.

At some point I started having an issue where this style block would randomly fail to initialize. No errors were reported.

I came across the "solution" by accident, when removing a big chunk of inner CSS would suddenly made it work. Then I tested it by splitting this one style element into two - also made it work.

Bottom line: a large-ish ( 1500 lines, generously spaced ? ) `custom-style` element fails to initialize ( still appears in DOM, with all its includes, but nothing else ).

**Comment by kirilledelman** [comment]
Could this also be caused by a CSS parsing error? Missing semicolon or something.. I haven't found one, but it hit me as a possibility for this scenario as well.

**Comment by kirilledelman** [comment]
Getting this in some other places as well - namely a large <style> inside <dom-module> of a component.

Splitting into two style sheets fixes it. Could be some internal Chrome problem?

**Comment by kirilledelman** [comment]
Seriously, this happens a lot - styles fail to initialize.

**Comment by TimvdLippe** [comment]
Given the age of this issue and no runnable JSBin, I am going to close this issue. If you can still reproduce this issue in Polymer 2 with the polyfills, please reopen this issue with a JSBin that shows the issue.
